.github/workflows/build.yml

```diff
@@ -191,9 +191,9 @@ jobs:
           rustup-components: rust-std rustfmt # Keep them in one line due to https://github.com/PyO3/maturin-action/issues/153
           args: --release --manylinux 2014 --features protoc,substrait
       - name: Archive wheels
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: dist
+          name: dist-manylinux
```

Comment:
nit: `dist-manylinux-x86_64` to match pattern for aarch64

Reply:
I additionally updated the job name for consistency `build-manylinux` --> `build-manylinux-x86_64`